src/main/docker/env.bash

```diff
@@ -13,5 +13,5 @@ export REMREM_GENERATE_PORT=8095
 export REMREM_PUBLISH_PORT=8096
 
 export REMREM_PUBLISH_RABBITMQ_INSTANCES_LIST="[\
-{ \"mp\": \"eiffelsemantics\", \"host\": \"rabbitmq\", \"port\": \"5672\", \"username\": \"myuser\", \"password\": \"myuser\", \"tls\": \"\", \"exchangeName\": \"ei-exchange\", \"domainId\": \"ei-domain\", \"createExchangeIfNotExisting\":true }\
+{ \"mp\": \"eiffelsemantics\", \"host\": \"rabbitmq\", \"port\": \"5672\", \"username\": \"myuser\", \"password\": \"myuser\", \"tls\": \"\", \"exchangeName\": \"eiffel-exchange\", \"domainId\": \"ei-domain\", \"createExchangeIfNotExisting\":true }\
```

Comment:
plase change ri-domain to gerrit-test-domain or something.

Reply:
now it is "gerrit-test-domain".  In next commit